File: lab6.cpp
#include <iostream>
#include <math.h>

using namespace std;
 
float a, b, c;
float t, s;
float L;
float funct(float, float, float);

void init();
void solution();
void browse();

 int main(){
     cout <<" \n" <<"обчіслення L:\n" << "=============" << endl;

    init();

    solution();

    browse();

 }
 float f(float a, float b, float c){
     return ((2 * a - b - (sin(c))) / (5 + abs(c)));
 }
 void init(){
     cout << "введіть t: "; cin >> t;
     cout << "введіть s: "; cin >> s;
 }
void solution(){
    L= f(t, -2 * s, 1.17) + f(2.2, t, s-t);
}
void browse(){
    cout << "L= " << L;
}
 

File: lab6.py
from math import exp
import math
print(str(" computation L: "))
print(str("==============="))
def fun(a, b, c):
    return ((2 * a - b - (math.sin(c) )) / (5 + abs(c)))

t = float(input("t = "))
s = float(input("s = "))

L=fun(t, -2*s, 1.17)+fun(2.2, t, s-t)

print (L)

